plugins/base/src/main/resources/dokka/scripts/symbol-parameters-wrapper_deferred.js

```diff
@@ -1,83 +1,57 @@
 // helps with some corner cases where <wbr> starts working already,
 // but the signature is not yet long enough to be wrapped
-const leftPaddingPx = 60
-
-const symbolResizeObserver = new ResizeObserver(entries => {
-    entries.forEach(entry => {
-        const symbolElement = entry.target
-        symbolResizeObserver.unobserve(symbolElement) // only need it once, otherwise will be executed multiple times
-        wrapSymbolParameters(symbolElement);
-    })
-});
-
-const wrapAllSymbolParameters = () => {
-    document.querySelectorAll("div.symbol").forEach(symbol => wrapSymbolParameters(symbol))
-}
-
-const wrapSymbolParameters = (symbol) => {
-    let parametersBlock = symbol.querySelector("span.parameters")
-    if (parametersBlock == null) {
-        return // nothing to wrap
-    }
-
-    let symbolBlockWidth = symbol.clientWidth
-
-    // Even though the script is marked as `defer` and we wait for `DOMContentLoaded` event,
-    // it can happen that `symbolBlockWidth` is 0, indicating that something hasn't been loaded.
-    // In this case, just retry once all styles have been applied and it has been resized correctly.
-    if (symbolBlockWidth === 0) {
-        symbolResizeObserver.observe(symbol)
-        return
+(function() {
+    const leftPaddingPx = 60;
+
+    function createNbspIndent () {
+        let indent = document.createElement("span")
+        indent.append(document.createTextNode("\u00A0\u00A0\u00A0\u00A0"))
+        indent.classList.add("nbsp-indent")
+        return indent
     }
 
-    let innerTextWidth = Array.from(symbol.children)
-        .filter(it => !it.classList.contains("block")) // blocks are usually on their own (like annotations), so ignore it
-        .map(it => it.getBoundingClientRect().width).reduce((a, b) => a + b, 0)
-
-    // if signature text takes up more than a single line, wrap params for readability
-    let shouldWrapParams = innerTextWidth > (symbolBlockWidth - leftPaddingPx)
-    if (shouldWrapParams) {
-        parametersBlock.classList.add("wrapped")
-        parametersBlock.querySelectorAll("span.parameter").forEach(param => {
-            // has to be a physical indent so that it can be copied. styles like
-            // paddings and `::before { content: "    " }` do not work for that
-            param.prepend(createNbspIndent())
-        })
+    function wrapSymbolParameters(entry) {
+        const symbol = entry.target;
+        const futureWidth = entry.borderBoxSize && entry.borderBoxSize[0] && entry.borderBoxSize[0].inlineSize;
```

Comment:
Not sure I understand this line fully. Why is it called `**future**Width`? Is it not the current width of the entry?

Reply:
As i know, resize observer handler happens on *layout* step of browser rendering. On this step all node sizes have been calculated but don't showed on screen, cause it may modified in handler. And, technically, it's future width. But in our case it isn't necessary information and over complicated. I returned old name `symbolBlockWidth`

Reply:
ooh, ok, after you explained it - `futureWidth` makes total sense :)